notebooks/community/feature_store/mobile_gaming_feature_store.ipynb

```diff
@@ -0,0 +1,2211 @@
+{
```

Comment:
<p>Consider explaining why this flag is turned on</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
Added comment